docs/development/workflow/development_workflow.rst

```diff
@@ -144,7 +144,9 @@ Astropy Guidelines for `git`_
   changes in the development version require changes to our code you can
   :ref:`rebase`.
 * If you need to edit `.mailmap <https://git-scm.com/docs/gitmailmap>`_ and
-  know how to do it then you can open a pull request for that.
+  know how to do it then you can open a pull request for that. Please run
+  ``git shortlog -ens`` locally first with your changes to make sure your
```

Comment:
It would be good to provide a link to the `git shortlog` documentation. Furthermore, I'm not sure why the `-n` flag would be helpful. It orders the output by the number of commits made by an author, but without it the output would be ordered by author name instead.
```suggestion
  `git shortlog -es <https://git-scm.com/docs/git-shortlog>`_ locally
  first with your changes to make sure your
```
That being said I don't think this commit should be in this pull request because it will complicate backporting the `.mailmap` edit.

Reply:
@eerovaher , is the `n` unnecessary?

As for backport... I'll deal with it if there is conflict.

Reply:
Ah, I missed your comment on `n` earlier, sorry. Hmm. I guess you are right.